src/components/SortIcon.tsx

```diff
@@ -0,0 +1,73 @@
+import { SortDirection } from 'react-stately';
+import styled, { css } from 'styled-components';
+
+import { Icon, IconName } from '@/components/Icon';
+
+type ElementProps = {
+  sortDirection: SortDirection | 'none';
+};
+
+type StyleProps = {
+  className?: string;
+};
+
+export const SortIcon = ({ sortDirection, className }: ElementProps & StyleProps) => {
```

Comment:
coool i dig this! superrrr nit since in the preview link i feel like they look a little too cartoonish(?) -- can we align it more like the one in figma so the two vertical lines of the arrows are closer to same height?
<img width="35" alt="image" src="https://github.com/user-attachments/assets/16ff6714-4e76-4aef-b8c1-2b01823886d0"><img width="32" alt="image" src="https://github.com/user-attachments/assets/07ba3633-85d0-4644-b05d-670a8af04760">

Reply:
ah yes - lemme update with achal's icons!